src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistPage.tsx

```diff
@@ -0,0 +1,159 @@
+import { Box, Button, Heading, Text } from 'grommet';
+import { Spinner } from 'lib-components';
+import { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { ReactComponent as CheckListIcon } from 'assets/svg/iko_checklistsvg.svg';
+import { SortableTable } from 'components/SortableTable';
+import { WhiteCard } from 'components/WhiteCard/WhiteCard';
+
+import { usePlaylists } from '../api/usePlaylists';
+
+const messages = defineMessages({
+  title: {
+    defaultMessage: 'My Playlists',
+    description: 'Playlist page title',
+    id: 'features.Playlist.title',
+  },
+  create: {
+    defaultMessage: 'Create playlist',
+    description: 'Create playlist button title',
+    id: 'features.Playlist.create',
+  },
+  noPlaylists: {
+    defaultMessage: 'You have no playlist yet.',
+    description: 'Message displayed when the user has not playlist yet.',
+    id: 'features.Playlist.noPlaylists',
+  },
+  tableTitle: {
+    defaultMessage:
+      '{item_count, plural, =0 {no playlist} one {# playlist} other {# playlists}}',
+    description: 'Playlist table title.',
+    id: 'features.Playlist.tableTitle',
+  },
+  error: {
+    defaultMessage: 'An error occurred, please try again later.',
+    description: 'Error message on loading playlists',
+    id: 'features.Playlist.error',
+  },
+  retry: {
+    defaultMessage: 'Retry',
+    description: 'Retry button title',
+    id: 'features.Playlist.retry',
+  },
+});
+
+export const PlaylistPage = () => {
+  const sorts = [{ label: 'by name' }, { label: 'by url' }];
```

Comment:
This is pure placeholder since their is not sort in the api for now (and I don't know what sorts we want XD)

Reply:
Their is no issue with that since the sorter is a templace that only require a label to display in the sort menu. But you can provide whatever you want

Reply:
My point is that bellow, in 
```
              onSortChange={(newSort) => {
                setCurrentSort(newSort);
                return data.results;
              }}
```
the label will be used (`newSort`), so if the label is a translation, you will have to use the translation to add logic.

Reply:
I'm not sure to understant, `newSort` is not just a label, it is the entire object selected in the menu.
So if this object contains an enum (and it probably should) you can use this enum to do your logic

Reply:
My bad, I don't know why I thougth I saw `newSort` was the label...